docs/manual-EN/2.query-language/0.README.md

```diff
@@ -1,3 +1,66 @@
 # Reader
 
 This chapter is for those who want to use `Nebula Graph` query language.
+
+## Placeholder Identifiers and Values
+
+The query language of **Nebula Graph**, i.e. the nGQL, refers to the following standards in design:
+
+- ISO/IEC 10646
+- ISO/IEC 39075
+- ISO/IEC NP 39075 (Draft)
+
+In template code, any token that is not a keyword, a literal value, or punctuation is a placeholder identifier or a placeholder value.
+
+Tokens details please refer to the following table:
+
+|  Token | Meaning  |
+|  ----  | ----  |
+| < >    | name of a syntactic element |
+| ::=    | formula that defines an element |
+| [ ]    | optional elements |
+| { }    | explicitly specified elements |
+|  \|    | complete alternative elements |
+| ...    |  may be repeated any number of times |
+
+Example:
+
+```bash
+INSERT VERTEX <tag_name> [, <tag_name>, ...] ( <prop_name_list>[, <prop_name_list> ...])
+     {VALUES | VALUE} <vid>: ( <prop_value_list> [, <prop_value_list> ...])
+
+prop_name_list:
+  [prop_name [, prop_name] ...]
+
+prop_value_list:
+  [prop_value [, prop_value] ...]
+```
+
+The user-defined identifiers are tag_name, prop_name_list and prop_value_list.
+
+- Choices
+
+    The vertical bar `|` is used to separate the choices, when the syntax requires that the user choose one out of a set of values. Example:  Either the keyword TAG or EDGE is to be used.
```

Comment:
The vertical bar `|` is used to separate options, which indicates that you have to choose one item from the option set. Example:  You have to choose one item from the keywords TAG and EDGE.